### PR TITLE
Crossover designer vignette

### PR DESCRIPTION
**Pull request by jaspercooper**
Here is the crossover vignette. I added systemfit as a "suggestion" to the package as we really need SUR for this designer. Rather than have it inside the designer the vignette just adds it to the design. 

In general if a package is essential for a given designer we should add it to suggests and keep it out of the designer, so that it can be used in the vignettes but won't mess with the rest of the package too much.

### CODE REVIEW

**Comment by jaspercooper** [comment]
Thanks for the review!

Re description: In general I think it is a good rule for them to be consistent but not identical. For one thing we need a description of the parameters in Shiny (not in documentation), and we also need html.

**Comment by clabima** [comment]
I agree. But shouldn't we also shoot for consistency across designers? For
eg. simple_two_arm_designer() currently does reference function parameters.

Em qua, 27 de jun de 2018 às 11:22, Jasper Cooper <notifications@github.com>
escreveu:

**Comment by jaspercooper** [comment]
Yes.

**Comment by macartan** [comment]
I think making reference to the designers and how to use these makes a lot
of sense. Reading vignettes I think a natural reaction is OK sounds good,
now how do I do it. Maybe there should be a generic box to get people going
from scratch:

To make a X design like this:
```
1. install declaredesign ...
2, install design library ...
3.  type: design <- my_X_designer(N = 100)
```
For a full list of arguments you can use with this designer  see here or in
R type:
```
? my_X_designer
```

**Comment by jaspercooper** [comment]
I like this idea a lot. Does someone want to have a go at including something like this in one of the vignettes and the rest of us can update accordingly on our respective vignettes?

**Comment by clabima** [comment]
Sure, can do as I revise. I'm thinking a "Getting started" at the very
bottom, unless there are other suggestions?

Em qua, 27 de jun de 2018 às 12:03, Jasper Cooper <notifications@github.com>
escreveu:

**Comment by jaspercooper** [comment]
Sorry commented on wrong thread lol

Let's call it "Using the X designer" and I agree at very bottom. Thanks so much!

**Comment by clabima** [comment]
Added template to pretest_posttest_designer vignette: de560db.

File: vignettes/crossover.Rmd [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+The diagnosis reveals a bias-variance tradeoff. Here, the no crossover effect assumption is violated. This causes a small amount of bias in the SUR estimator. However, even a small amount of correlation in the errors --- $\rho = .2$ --- significantly increases the efficiency of the SUR estimator here. Thus, in terms of RMSE the bias is offset by the increased power of the estimator. 
+
+### Using the Crossover Designer
+
+In R, you can generate a crossover design using the template function `crossover_designer()` in the `DesignLibrary` package by running the following lines, which install and load the package:
+
+```{r, eval=FALSE}
+install.packages("DesignLibrary")
```
Comment (jaspercooper) [inline vignettes/crossover.Rmd]: @clabima this should be 
`devtools::install_github("DeclareDesign/DesignLibrary", keep_source = TRUE)`
in all vignettes

**Pull request merged**